community/wagtail_hooks.py

```diff
@@ -190,25 +186,25 @@ class MeetingDocumentModelAdmin(ModelAdmin):
     )
 
 
-class CommunityGroup(ModelAdminGroup):
+class CommunityGroup(ModelViewSetGroup):
     menu_label = "Community"
     menu_icon = "snippet"
     menu_order = 200
     items = (
-        PersonModelAdmin,
-        MeetingModelAdmin,
-        OrganizationModelAdmin,
-        EventModelAdmin,
-        MemorialModelAdmin,
-        CommunityDirectoryModelAdmin,
-        OnlineWorshipModelAdmin,
-        PublicBoardDocumentModelAdmin,
-        MeetingDocumentModelAdmin,
-        MollyWingateBlogPageModelAdmin,
+        PersonViewSet,
+        MeetingViewSet,
+        OrganizationViewSet,
+        EventViewSet,
+        MemorialViewSet,
+        CommunityDirectoryViewSet,
+        OnlineWorshipViewSet,
+        PublicBoardDocumentViewSet,
+        MeetingDocumentViewSet,
+        MollyWingateBlogPageViewSet,
     )
 
 
-modeladmin_register(CommunityGroup)
+hooks.register("register_community_viewset", CommunityGroup)
```

Comment:
I believe we need to prefix hooks with @ 
```suggestion
@hooks.register("register_community_viewset", CommunityGroup)
```

Otherwise, the menus aren't showing up.